docs/new-architecture-intro.md

```diff
@@ -24,10 +24,8 @@ The guide is divided into five sections:
   - [Prerequisites for Supporting the New Architecture in your App](new-architecture-app-intro)
   - Enabling the New NativeModule System (TurboModule) in your App
     - [Android](new-architecture-app-modules-android)
-    - [iOS](new-architecture-app-modules-ios)
   - Enabling the New Renderer (Fabric) in your App
     - [Android](new-architecture-app-renderer-android)
```

Comment:
This looks unbalanced now. Should we just have somehting like:
```
- Android
  - Enabling the New NativeModule System (TurboModule) in your App
  - Enabling the New Renderer (Fabric) in your App
```

Reply:
Ops... adressed in #3312